File: threedollar-api-user/src/main/java/com/depromeet/threedollar/api/user/service/auth/AuthService.java
package com.depromeet.threedollar.api.user.service.auth;

import com.depromeet.threedollar.api.user.service.auth.dto.request.LoginRequest;
import com.depromeet.threedollar.api.user.service.auth.dto.request.SignUpRequest;

public interface AuthService {

    Long signUp(SignUpRequest request);

    Long login(LoginRequest request);

}

File: threedollar-api-boss-service/src/main/kotlin/com/depromeet/threedollar/api/bossservice/service/auth/AuthService.kt
package com.depromeet.threedollar.api.bossservice.service.auth

import com.depromeet.threedollar.api.bossservice.service.auth.dto.request.LoginRequest

interface AuthService {

    fun login(request: LoginRequest): String

    fun getSocialId(token: String): String

}
